scripts/killall_sglang.sh

```diff
@@ -8,7 +8,6 @@ kill -9 $(ps aux | grep 'sglang::' | grep -v 'grep' | awk '{print $2}') 2>/dev/n
 kill -9 $(ps aux | grep 'sglang.launch_server' | grep -v 'grep' | awk '{print $2}') 2>/dev/null
 kill -9 $(ps aux | grep 'sglang.bench' | grep -v 'grep' | awk '{print $2}') 2>/dev/null
 
-# Clean all GPU processes if any argument is provided
-if [ $# -gt 0 ]; then
-    kill -9 $(nvidia-smi | sed -n '/Processes:/,$p' | grep "   [0-9]" | awk '{print $5}') 2>/dev/null
-fi
+# Clean all GPU processes
+kill -9 $(nvidia-smi | sed -n '/Processes:/,$p' | grep "   [0-9]" | awk '{print $5}') 2>/dev/null
+lsof /dev/nvidia* | awk '{print $2}' | xargs kill -9 2>/dev/null
```

Comment:
can we show nvidia-smi again at the end to ensure everything is cleaned up? seems that 
<img width="907" alt="image" src="https://github.com/user-attachments/assets/6b878688-5424-4ce4-8a88-4fdd1c68ce91" />
still has issues

Reply:
this one is from another PR which doesn't contain this fix tho. updated the script to show gpu status again.

Reply:
ata is fast :rocket: